docs/development/architecture/PACKAGES.md

```diff
@@ -7,12 +7,61 @@ sidebar_label: Packages
 This page describes the top-level packages exported by the [typescript-eslint monorepo](https://github.com/typescript-eslint/typescript-eslint).
 Each of these are published as npm packages under the `@typescript-eslint` organization.
 
-## `@typescript-eslint/eslint-plugin`
+```mermaid
```

Comment:
Related: https://github.com/facebook/docusaurus/pull/7490

I'm positive that one can be merged soon, provided it gets timely reviews